app/models/game.rb

```diff
@@ -1,4 +1,52 @@
 class Game < ActiveRecord::Base
   has_many :pieces
   belongs_to :user
+  after_create :populate_board!
+
+  def populate_board!
+    init_pawn
+    init_rook
+    init_knight
+    init_bishop
+    init_queen
+    init_king
+  end
+
```

Comment:
maybe make the separate piece init methods private? prevent people from trying to run them themselves. Let me know your thoughts